src/braintree-manager/payment-providers/apple-pay/apple-pay-session-manager.ts

```diff
@@ -0,0 +1,21 @@
+export interface ApplePaySessionManagerInterface {
+  canMakePayments(): boolean;
+  createNewPaymentSession(paymentRequest: ApplePayJS.ApplePayPaymentRequest): ApplePaySession;
+}
+
+export class ApplePaySessionManager implements ApplePaySessionManagerInterface {
+  static VERSION = 3;
```

Comment:
why do we need to be set to this version? just in case?

Reply:
This sets the API version needed to process ApplePay payments... down below in `canMakePayments()`, we verify the browser can support this version and when we create a new payment session, we initialize it with this version so their systems know which version to use.